db_stress_tool/no_batched_ops_stress.cc

```diff
@@ -621,9 +634,12 @@ class NonBatchedOpsStressTest : public StressTest {
           switch (op) {
             case 0:
             case 1: {
-              const uint32_t value_base = 0;
+              ExpectedValue put_value(thread->rand.Next());
```

Comment:
Technically we want `ExpectedValue` to start with an valid ExpectedValue rather than just an arbitrary number. The reason is internal detail: the pending write bit and delete bit of ExpectedValue shouldn't be set the same time other than during the short period of time after crash-recovery but before crash-recovery-verification/remediation. 

However you as a user of `ExpectedValue` shouldn't be concerned about this detail so it's more of my API design gap. 

With that, would you mind adding a default constructor of `ExpectedValue` taking no parameter and set `expected_value_` to be `DEL_MASK`? Basically it means this expected value starts with "being deleted". So in `case 1: ` we put a new value; in `case 2` we will attempt to `delete_value.Delete()` but the internal details will just ignore the delete (cuz it's already deleted), which as a user you wouldn't need to care about.

Reply:
> With that, would you mind adding a default constructor of ExpectedValue taking no parameter and set expected_value_ to be DEL_MASK and use that constructor instead?

I kind of felt like this was also one of the suggestions you mentioned in our 1:1 meeting - was it??

Reply:
Thank you for the suggestion. I added the constructor you suggested. I agree that randomness is not a must for what this verification does.

Reply:
Seems like this suggestion still needs to be resolved by using the default constructor in this version of code.